### DISCUSSION TITLE
Path to JAVA DB downloaded when generating SBOM

### DISCUSSION THREAD

**Discussion by milairhu** [aquasecurity/trivy#7102] [Q&A] [answered]
### Question

Hi everyone,

I can't find where the JAVA DB is stored when running Trivy to generate SBOMs. 
Do you have the response? And how did you find it out?

Thanks,
Hugo

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

**Comment by DmitriyLewen** [comment, marked as answer]
Hello @milairhu 

path is `<cache_dir>/trivy/java-db`
https://aquasecurity.github.io/trivy/v0.53/docs/coverage/language/java/#jarwarparear

**Reply by milairhu** [reply]
Thank yout very much!
